static-src/scripts/App.js

```diff
@@ -30,7 +27,5 @@ module.exports = App.extend({
       hashChange: true
     });
     */
-
-  }
-
+  },
```

Comment:
That might be a good rule to not use. Those trailing commas are going to drive me crazy ;-)